profile.graphqls

```diff
@@ -156,6 +163,10 @@ extend type Query {
     getProfileTaskSegmentList(taskID: String): [BasicTrace!]!
```

Comment:
@mrproliu How about UI enhancing this way,

1. Use this API, to group `segmentId`s by `traceId` and `Instance ID`(new Added). Could our backend these without agent update?
2. `getProfiledSegments`(new APIs) to get `ProfiledSegment`s. 
3. Build profiling.

The major differences with yours are
1. `getProfiledSegment` should be `deprecated`, only a shell(special condition) to `getProfiledSegments`.
2. No new `ProfiledSegments`.